packages/material-ui/src/ToggleButton/ToggleButton.js

```diff
@@ -215,7 +215,7 @@ ToggleButton.propTypes /* remove-proptypes */ = {
    * @default 'medium'
    */
   size: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([
-    PropTypes.oneOf(['standard', 'primary', 'secondary', 'error', 'info', 'success', 'warning']),
+    PropTypes.oneOf(['large', 'medium', 'small']),
```

Comment:
I will send a PR to reverse this order, we can use logical order now that it's not generated from the types

```suggestion
    PropTypes.oneOf(['small', 'medium', 'large']),
```